pkg/sylog/sylog_dummy_test.go

```diff
@@ -25,7 +25,7 @@ func TestGetLevel(t *testing.T) {
 	defer test.ResetPrivilege(t)
 
 	l := GetLevel()
-	if l != -1 {
+	if l != 1 {
 		t.Fatalf("%d was returned instead of -1", l)
```

Comment:
The message now does not match the check.

Reply:
got it!
reflected here
https://github.com/apptainer/apptainer/pull/1472/commits/157b946964534ab2b1cb139b399bf3b4fc8893b5